packages/paste-core/components/badge/src/styles.ts

```diff
@@ -20,19 +24,93 @@ export const badgeVariantStyles: {
     backgroundColor: 'colorBackgroundNew',
     color: 'colorTextNew',
   },
-  info: {
+  neutral: {
     backgroundColor: 'colorBackgroundNeutralWeakest',
     color: 'colorTextNeutral',
   },
-  default: {
-    backgroundColor: 'colorBackground',
-    color: 'colorTextWeak',
+  decorative10: {
+    backgroundColor: 'colorBackgroundDecorative10Weakest',
+    color: 'colorTextDecorative10',
+  },
+  decorative20: {
+    backgroundColor: 'colorBackgroundDecorative20Weakest',
+    color: 'colorTextDecorative20',
+  },
+  decorative30: {
+    backgroundColor: 'colorBackgroundDecorative30Weakest',
+    color: 'colorTextDecorative30',
+  },
+  decorative40: {
+    backgroundColor: 'colorBackgroundDecorative40Weakest',
+    color: 'colorTextDecorative40',
+  },
+  neutral_counter: {
+    backgroundColor: 'colorBackgroundNeutralWeakest',
+    color: 'colorTextNeutral',
+    borderRadius: 'borderRadiusPill',
+  },
+  error_counter: {
+    backgroundColor: 'colorBackgroundErrorWeakest',
+    color: 'colorTextError',
```

Comment:
```suggestion
    color: 'colorTextErrorStrong',
```

Reply:
Can we update both the error Badge and the error counter Badge to use colorTextErrorStrong?

Reply:
yes yes, big fan. Done.